packages/react-dev-utils/ModuleScopePlugin.js

```diff
@@ -13,8 +13,9 @@ const chalk = require('chalk');
 const path = require('path');
 
 class ModuleScopePlugin {
-  constructor(appSrc) {
+  constructor(appSrc, allowedPaths = []) {
```

Comment:
Small nit, how about `allowedFiles`? Paths makes it sound like you can give it a directory.